NEWS.md

```diff
@@ -49,6 +52,9 @@ Standard library changes
 * A no-argument construct to `Ptr{T}` has been added which constructs a null pointer ([#30919])
 * `strip` now accepts a function argument in the same manner as `lstrip` and `rstrip` ([#31211])
 * `mktempdir` now accepts a `prefix` keyword argument to customize the file name ([#31230], [#22922])
+* `nextfloat(::BigFloat)` and `prevfloat(::BigFloat)` now returns a value with the same precision
+    as their argument, which means that (in particular) `nextfloat(prevfloat(x)) == x` whereas
+    previously this could result in a completely different value with a different precision ([#31310])
```

Comment:
```suggestion
  previously this could result in a completely different value with a different precision ([#31310])
```

Reply:
It is just a markdown thing. I had `build` the docs and checked that it looks fine. Do you still want me to push to these changes?

Reply:
Probably not worth it, although it does match the way the rest of the file is formatted. You can also just merge the suggested changes through the GitHub UI.

Reply:
Changed the files.